Package.swift

```diff
@@ -4,14 +4,14 @@ import PackageDescription
 let package = Package(
     name: "Mockolo",
     platforms: [
-        .macOS(.v10_15),
+        .macOS(.v12),
     ],
     products: [
         .executable(name: "mockolo", targets: ["Mockolo"]),
         .library(name: "MockoloFramework", targets: ["MockoloFramework"]),
-        ],
+    ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-syntax.git", exact: "509.0.0"),
+        .package(url: "https://github.com/apple/swift-syntax.git", from: "510.0.1"),
```

Comment:
Hey there, 
I think it's better to lock to a specific version of swift-syntax to avoid issues with transient dependencies.

Reply:
That's an old style. Please take a look at the latest swift-syntax.